.husky/commit-msg

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
```

Comment:
That file supposed to be generated locally/present in 3rd party project, right?

Reply:
@vadim-c-noveo Yes, when running `npx husky install` (which is executed via the prepare script) the `_` folder and contents are created by Husky.

Reply:
P.S. And the prepare script is executed when calling `npm install/ci/i`